### PR TITLE
CHAT-70 React unit tests with Jest and Enzyme - initial

### PR DESCRIPTION
**Pull request by Charleswmm**

### CODE REVIEW

File: src/scss/components/MainNavButton/MainNavButton.scss [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
   .pip {
     position: absolute;
-    top: 14px;
+    top: 13px;
```
Comment (jakxnz) [inline src/scss/components/MainNavButton/MainNavButton.scss:17]: I'm assuming these are `px` for good reason?

File: src/js/components/MainNavButton/MainNavButton.test.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+      }
+    }
+
+    const wrapper = mount(
+      <GlobalContext.Provider value={ values } >
+        <MainNavButton imageSrc={ 'foo' } />
+      </GlobalContext.Provider>
+    );
+
+    expect(wrapper.find('.nav-channel').prop('style')).toHaveProperty('backgroundImage', "url(foo)")
```
Comment (jakxnz) [inline src/js/components/MainNavButton/MainNavButton.test.js:59]: Nice

**Review by jakxnz — APPROVED** [review]
LGTM

Those tests are really concise. Nice work

File: src/scss/components/MainNavButton/MainNavButton.scss (hunk 1 above)
Comment (Charleswmm) [reply src/scss/components/MainNavButton/MainNavButton.scss:17]: Im not sure, there might have been. I saw it wasn't aligning properly, so I fixed it

**Pull request merged**